config.js

```diff
@@ -84,6 +84,12 @@ const schema = {
     default: null,
     env: 'GITHUB_TOKEN'
   },
+  githubWebhookSecret: {
```

Comment:
After thinking about this further, and [digging into the GH docs a bit](https://developer.github.com/webhooks/securing/) I realize that I had initially missed that this webhook secret is set on a per/repo basis. 

With that in mind, I think it makes more sense for those values to be set in siteConfig.js and read as part of the per-repo config. This will require the value be encrypted, but we already do that with other values in that config. Thoughts? I think this is an important distinction so that one Staticman instance can serve multiple projects and/or users.

Reply:
Good catch. I just sort of followed along with the precedent set by `githubToken` being in the JSON config. However, that is for the bot, which can be shared. Might be worth considering renaming that property to `githubBotToken`. Anyway, I'll work on moving both secrets to the YAML site config. Thanks.

Reply:
OK, the configuration of the expected webhook secrets have been moved from the JSON `config` to the YAML `siteConfig`. I updated the unit tests and manually tested on Heroku against GitHub (endpoints v1 and v3) and GitLab (v3), using various permutations of secret configurations. Review away. :)